src/Learning/SFA.DAS.Learning.Api.UnitTests/Controllers/CollectionCalendar/WhenGettingAcademicYear.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.Learning.Api.Controllers;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests.CollectionCalendar;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.CollectionCalendar;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Learning.Api.UnitTests.Controllers.CollectionCalendar;
+
+[TestFixture]
+public class WhenGettingAcademicYear
+{
+    [Test, MoqAutoData]
+    public async Task Then_Gets_ApprenticeshipKey_From_ApiClient(
```

Comment:
I know this is maybe part of the inner rename, but this should be LearningKey now